.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: node_js
```

Comment:
Are we sure we want to use travis for this? I would think we would want something like github actions instead

Reply:
I chose this because currently all of our lambdas deploy using travis. So we could change to github actions, but if we do I think we should do all of them at once to be consistent.